modules/core/shared/src/main/scala/lucuma/core/math/Angle.scala

```diff
@@ -153,13 +151,22 @@ sealed class Angle protected (val toMicroarcseconds: Long) {
     toMicroarcseconds.toInt
 
   /**
-   * angle difference or separation between two angles in the range [0 .. π]
-   */
+    * angle difference or separation between two angles in the range [0 .. π]
+    */
   def difference(a: Angle): Angle =
     Angle.difference(this, a)
 }
 
 object Angle extends AngleOptics {
+  type Angle180µas = 648000000000L  // 180 * µasPerDegree
```

Comment:
Why a type?

Reply:
It is used in `type Parallaxμas = Interval.Closed[0, Angle.Angle180µas]` in Parallax, where it has to be a type.

Reply:
Oh neat

Reply:
Thinking about it a bit more, it might make more sense to say `final val Angle180µas = 648000000000L` and then use `Interval.Closed[0, Angle.Angle180µas.type]` so we have both the type and term available.

Reply:
We do have both available, because I create `µasPer180` using `valueOf[Angle180µas]` a few lines down. But, it would be terser to use `.type` where needed. I can switch to that. Is the `final` needed?

Reply:
Your solution with `valueOf` is fine, I don't feel strongly. 

But yes, `final` is required to get a singleton type.

```scala
scala> object Foo { val x = 123 }
object Foo

scala> val y: Foo.x.type = 42
                           ^
       error: type mismatch;
        found   : Int(42)
        required: Foo.x.type

scala> val y: Foo.x.type = 123
                           ^
       error: type mismatch;
        found   : Int(123)
        required: Foo.x.type

scala> object Foo { final val x = 123 } 
object Foo

scala> val y: Foo.x.type = 42
                           ^
       error: type mismatch;
        found   : Int(42)
        required: 123

scala> val y: Foo.x.type = 123 
val y: 123 = 123
```

Reply:
Interesting. I can do `Interval.Closed[0, Angle.µasPer180.type]` without µasPer180 being final and it seems to work fine. 🤔 I was a bit surprised that I could also use `val µasPer180 = 180L * µasPerDegree` and that worked.

Reply:
I think you won't be able to construct a value of that type with that bound unless it's final. Kind of error-prone, I think I like the `valueOf` strategy better now.

Reply:
OK. I will leave it as it is.

Reply:
Just one last bit that is interesting (to me). With your first version of Foo with the non-final val, I can't do `val xx: Foo.x.type = 123`, but I CAN do `val xx: Int = valueOf[Foo.x.type]`.